packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js

```diff
@@ -0,0 +1,123 @@
+import first from 'lodash/first';
+
+export default class PciUsersAddController {
+  /* @ngInject */
+  constructor(
+    $translate,
+    CucCloudMessage,
+    PciStoragesObjectStorageService,
+    $state,
+  ) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciStoragesObjectStorageService = PciStoragesObjectStorageService;
+    this.$state = $state;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+    this.disable = true;
+    this.addExistingUser = 'addExistingUser';
+    this.createNewUser = 'createNewUser';
+    this.allUserList = this.allUserList.map((userList) => ({
+      ...userList,
+      asCredentials:
+        this.isAsCredentials(this.usersCredentials, userList.id)
+          .asCredentials === true
```

Comment:
**suggestion:** Since `isAsCredentials` has a single statement. Better not to create a separate function 
```suggestion
        this.usersCredentials.find((credential) => credential.userId === userList.id). asCredentials
```